controllers/release_controller.go

```diff
@@ -74,9 +76,26 @@ func (r *ReleaseReconciler) triggerReleasePipeline(ctx context.Context, release
 		return ctrl.Result{}, nil
 	}
 
+	err = r.checkReleaseLinkTargetExists(ctx, releaseLink)
+	if err != nil {
+		log.Error(err, "Failed to find ReleaseLink's target workspace")
+		release.Status.SetErrorCondition(err)
+
+		return ctrl.Result{}, nil
+	}
+
+	_, err = r.getTargetReleaseLink(ctx, releaseLink)
```

Comment:
@davidmogar This line should be
`targetReleaseLink, err := r.getTargetReleaseLink(ctx, releaseLink)`
but then it fails for unused variable (it won't be used until your PR). So you'll have to change to that line as part of your PR